docs/guides/unmanic_link_installations.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Linking Unmanic Installations
+description: Linking Unmanic Installations
+id: unmanic_link_installations
+---
+![Difficulty](https://img.shields.io/badge/Difficulty-moderate-orange?style=flat)
+![Setup Time](https://img.shields.io/badge/Setup%20Time-10%20minutes-orange?style=flat)
+[![Original Author](https://img.shields.io/badge/Original%20Author-fredrikbaberg-lightgrey?style=flat?style=plastic&logo=github)](https://github.com/fredrikbaberg)
+
+Multiple installations of Unmanic can be linked, this allows the distribution of tasks to be processed.
+
+This guide will describe the use of a main installation with access to the library, with a remote installation that does the work.
+
+## Instructions
+
+### Requirements
+
+* At least two installs of Unmanic:
+    * One install will be called `main`, this has access to a folder with files to process.
+    * The other install will be called `remote`, this has to be reachable from `main` but does not require access to the files. Files will automatically be transfered from `main` to `remote` and back.
+
+### Setup
+
+* On `main`, setup a [library](/docs/configuration/libraries/adding_libraries.mdx) following the normal instructions. Let us call the library `Guide`.
```

Comment:
Should this be changed to `[library](adding_libraries)`, and similar for other references?